src/book.js

```diff
@@ -157,42 +157,50 @@ export class Book {
     let pages;
     [this.managedDoc, pages] = await this.embedPagesInNewPdf(this.currentdoc);
 
-    for (var i = 0; i < pages.length; ++i) {
-      var page = pages[i];
-      var newPage = this.managedDoc.addPage();
-      var rotate90cw =
-        this.source_rotation == '90cw' ||
-        (this.source_rotation == 'out_binding' && i % 2 == 0) ||
-        (this.source_rotation == 'in_binding' && i % 2 == 1);
-      var rotate90ccw =
-        this.source_rotation == '90ccw' ||
-        (this.source_rotation == 'out_binding' && i % 2 == 1) ||
-        (this.source_rotation == 'in_binding' && i % 2 == 0);
-      if (this.source_rotation == 'none') {
-        newPage.setSize(page.width, page.height);
-        newPage.drawPage(page);
-      } else if (rotate90ccw) {
-        newPage.setSize(page.height, page.width);
-        newPage.drawPage(page, {
-          x: page.height,
-          y: 0,
-          rotate: degrees(90),
-        });
-      } else if (rotate90cw) {
+    pages.forEach((page, i) => {
+      const newPage = this.managedDoc.addPage();
+      let rotateAngle = -1;
```

Comment:
This should probably be set to null (or undefined)

Reply:
set to null